api-docs/paths/events/issue-details.json

```diff
@@ -214,6 +214,28 @@
                 "type": "string",
                 "description": "The new status for the issues. Valid values are `\"resolved\"`, `\"reprocessing\"`, `\"unresolved\"`, and `\"ignored\"`."
               },
+              "statusDetails": {
```

Comment:
@sentaur-athena Do Sentry docs not render nested fields? Thought this would work but it did not, see: https://docs.sentry.io/api/events/update-an-issue/

<img width="560" alt="Screenshot 2024-08-28 at 2 16 42 PM" src="https://github.com/user-attachments/assets/0e5a064f-d767-4caa-914d-7aec89bed00f">